module/zfs/vdev_raidz.c

```diff
@@ -1440,6 +1440,94 @@ vdev_raidz_child_done(zio_t *zio)
 	rc->rc_skipped = 0;
 }
 
+
+/* Rotating media load calculation configuration. */
+extern int zfs_vdev_mirror_rotating_inc;
+extern int zfs_vdev_mirror_rotating_seek_inc;
+extern int zfs_vdev_mirror_rotating_seek_offset;
+
+/* Non-rotating media load calculation configuration. */
+extern int zfs_vdev_mirror_non_rotating_inc;
+extern int zfs_vdev_mirror_non_rotating_seek_inc;
```

Comment:
While I hate the idea of even more tunings it may make the most sense to add a new set of `zfs_vdev_raidz_*` options.  I can imagine a situation where a system has two pools, one mirrored and one raidz.  For example, an SSD mirrored root pool and a HDD based raidz primary store may need to be tuned differently.